packages/toast/src/components/Toast/Toast.stories.tsx

```diff
@@ -31,7 +31,8 @@ const Template: Story<StoryArgs> = (args) => {
   toast.warning(text, { autoClose: false });
   toast.error(text, { autoClose: false });
 
-  return <Toast />;
+  // no transition to avoid issues with Chromatic snapshots
+  return <Toast disableTransition />;
```

Comment:
May be worth [disabling auto close](https://fkhadra.github.io/react-toastify/autoClose#prevent-the-toast-from-closing-by-default) to so they don't disappear while the snapshot is taken 🤔

Reply:
Good idea!
Ref: [disable autoclose for story](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6959/commits/c4c625ea3451aef73a57146695f9ab3abf12625f)